### ISSUE TITLE
Doesn't support non-tensor output with sklearn-onnx in onnxruntime server

### ISSUE THREAD

**Issue by arkarn** [microsoft/onnxruntime#4294]
**Describe the bug**
A sklearn randomforest model converted to onnx with convert_sklearn() throws :
[error] MLValueToTensorProto() failed. Output name: output_probability. Error Message: Don't support Non-Tensor values

**Urgency**
There is a use-cased being blocked due to this issue. Timelines for model deployment in onnxruntime is suppose to be EOBW

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- ONNX Runtime installed from (source or binary): binary
- ONNX Runtime version: 1.0.0
- Python version: 3.7
- Visual Studio version (if applicable):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**To Reproduce**
Follow the steps till creating onnx model from sklearn randomforest given in official documentation: http://onnx.ai/sklearn-onnx/
Use the onnx model in onnxruntime for inference.

**Expected behavior**
The output_probability from sklearn model should be a tensor by default

**Screenshots**

**Additional context**
Add any other context about the problem here. If the issue is about a particular model, please share the model details as well to facilitate debugging.

**Comment by pranavsharma** [comment]
Yes, this was intentional at the time of design. Contributions are welcome.

**Comment by arkarn** [comment]
@pranavsharma  Does this mean currently we cannot run inference with any sklearn model (converted to onnx) on onnxRTServer? 
Even the latest version 1.3.0 doesn't support this?
What is the workaround to get sklearn model working on onnxRTServer?

**Comment by pranavsharma** [comment]
> @pranavsharma Does this mean currently we cannot run inference with any sklearn model (converted to onnx) on onnxRTServer?

If a model requires non-tensor inputs/outputs, it's not supported.

> 
> Is there any workaround to get sklearn model working on onnxRTServer?

No. 
onnxruntime server is in maintenance mode and we don't plan to actively contribute to it. We welcome external contributions though.
